scripts/build/package.js

```diff
@@ -27,7 +29,7 @@ const indexTemplate = fs.readFileSync(indexTemplateFile, UTF8);
 const iconObjectTemplate = fs.readFileSync(iconObjectTemplateFile, UTF8);
 
 const data = require(dataFile);
-const { getIconSlug, titleToSlug } = require("../utils.js");
+const { getIconSlug, slugToVariableName } = require("../utils.js");
```

Comment:
It's not really clear what should go where, but I think it would make more sense to put this (`slugToVariableName`) as a [local helper function](https://github.com/simple-icons/simple-icons/blob/d2bd9906d8a0a2dd2aef855dbd223bc3937f4ce6/scripts/build/package.js#L32). Provided you agree, could you make that change?

Reply:
Done